processor/k8sprocessor/doc.go

```diff
@@ -0,0 +1,113 @@
+// Copyright 2019 Omnition Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package k8sprocessor allow automatic tagging of spans with k8s metadata.
```

Comment:
nit: comments here are very good, but, perhaps better for a README.md - doesn't need to be changed in this PR.

Reply:
These show up on Go docs. I'll add a task to add user documentation as a read me.

Reply:
https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/97